test/e2e/common.go

```diff
@@ -24,3 +25,28 @@ func validateChanges(t *testing.T, respTable []ui.JSONUITableResp, expected []ma
 	require.Equalf(t, notesWaitTo, respTable[0].Notes[1], "Expected to see correct summary, but did not: '%s'", output)
 
 }
+
+func replaceAge(result []map[string]string) []map[string]string {
+	for i, row := range result {
+		if len(row["age"]) > 0 {
+			row["age"] = "<replaced>"
+		}
+		result[i] = row
+	}
+	return result
+}
+
+func replaceLastChangeAge(result []map[string]string) []map[string]string {
```

Comment:
If this function is used only in the test cases, then it shouldn't be part of the main code.